moped-editor/src/views/projects/projectView/ProjectComponents/mapSettings.js

```diff
@@ -41,13 +41,13 @@ export const fitBoundsOptions = {
   zoomToExtent: {
     maxZoom: 15,
     // accounting for fixed top bar
-    padding: 200,
+    padding: 50,
   },
   zoomToClickedComponent: {
     maxZoom: 19,
     padding: 200,
     duration: 1500,
-    linear: true
+    linear: true,
```

Comment:
@mddilley after clicking around a bunch with some projects, i'm leaning toward adopting the same padding for the `zoomToClickedComponent`. (for the zoom action that happens if you click the mag icon or save the component). it feels a little better/consistent especially when a project only has one component. i also think we could lower the zoom duration (but not make it too jerky). 

i would probably have a different opinion if you asked me tomorrow but i vote for these settings if others don't feel strongly about it:

```javascript
  zoomToClickedComponent: {
    maxZoom: 19,
    padding: 50,
    duration: 500,
    linear: true,
  }
```

Reply:
I like the shorter duration. It's enough motion to telegraph what is going on, but it gets it done quickly so the user can start using the map immediately.

Reply:
awesome! 🚢 🚢 🚢 🚢